### PR TITLE
Add which endpoints require authorization [SATURN-1669]

### PR DESCRIPTION
**Pull request by s-rubenstein**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ const main = async () => {
 
   /**
    * @api {post} /api/syncProfile Update mixpanel profile
-   * @apiDescription Syncs profile info from orchestration to mixpanel
+   * @apiDescription Syncs profile info from orchestration to mixpanel. Requires an authorization token to be set pull the profile from Orchestration.
```
Comment (ehanna4) [inline src/server.js]: I might be misreading this but I'm having a little trouble understanding this sentence in what is happening with Orchestration. Is the token pulling the profile from Orch?

Comment (s-rubenstein) [reply src/server.js]: That's a good question. From what I can tell, it is pulling the profile associated with a token, and syncing it into mixpanel. I don't know where we would use this though.

Comment (s-rubenstein) [reply src/server.js]: I will try a different wording.

**Review by ehanna4 — APPROVED** [review]
Looks good to me!

**Pull request merged**